docs/notebooks/batch_optimization.pct.py

```diff
@@ -64,13 +64,13 @@ def build_model(data):
 
 # %% [markdown]
 # ## Batch acquisition functions.
-# To perform batch BO, we must define a batch acquisition function. Three batch acquisition functions supported in Trieste are `BatchMonteCarloExpectedImprovement`, `LocalPenalizationAcquisitionFunction` and `GIBBON`.
+# To perform batch BO, we must define a batch acquisition function. Four batch acquisition functions supported in Trieste are `BatchMonteCarloExpectedImprovement`, `LocalPenalizationAcquisitionFunction`, `FantasizeAcquisitionFunction` and `GIBBON`.
```

Comment:
title contains only the original three, perhaps remove all of them and leave only "Batch Bayesian Optimization"?

Reply:
we should probably add references for these 4 approaches...